common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctions.scala

```diff
@@ -262,7 +262,7 @@ object SqlFunctions extends SqlFunctionsLocation with SqlFunctionsGeometry with
       case Some(op) =>
         foldSegments(sqlFragsAndParams._1.map(s => template.format(s)), op)
     }
-    ParametricSql(Seq(foldedSql), sqlFragsAndParams._2)
+    ParametricSql(foldedSql.split(SqlFragments.SeparatorRx).toSeq, sqlFragsAndParams._2)
```

Comment:
Newlines are allowed in our input... I'm not sure there's any sort of in-band separation that's safe to use.